includes/validation/class-amp-validation-manager.php

```diff
@@ -1946,15 +1927,15 @@ public static function unserialize_validation_error_messages( $serialized ) {
 	public static function get_validate_url_error_message( $error_code, $error_message = '' ) {
 		$check_error_log = sprintf(
 			/* translators: %s is link to Debugging in WordPress */
-			' ' . __( 'Please check your server PHP error logs; to do this you may need to <a href="%s" target="_blank">enable</a> <code>WP_DEBUG_LOG</code>.', 'amp' ),
+			__( 'Please check your server PHP error logs; to do this you may need to <a href="%s" target="_blank">enable</a> <code>WP_DEBUG_LOG</code>.', 'amp' ),
 			esc_url( 'https://wordpress.org/support/article/debugging-in-wordpress/' )
 		);
 
 		if ( $error_message ) {
-			$error_message = ' ' . rtrim( $error_message, '.' ) . '.';
+			$error_message = rtrim( $error_message, '.' ) . '.';
 		}
 
-		$support_forum_message = ' ' . sprintf(
+		$support_forum_message = sprintf(
 			/* translators: %1$s: Link to support forum. %2$s: Link to new topic form in support forum. */
 			__( 'If you are stuck, please search the <a href="%1$s">support forum</a> for possible related topics, or otherwise start a <a href="%2$s">new support topic</a> including the error message, the URL to your site, and your active theme/plugins.', 'amp' ),
```

Comment:
Translations should be escaped, as the actual HTML that will be sent to the browser is crowd-sourced user-input.

Reply:
See d48ceae6f50623a6e0abb63057f7dd294e98d54c